rinja/src/filters/builtin.rs

```diff
@@ -265,14 +264,13 @@ pub fn upper(s: impl fmt::Display) -> Result<String, fmt::Error> {
     upper(try_to_string(s)?)
 }
 
-/// Alias for the `upper()` filter
-/// Converts to uppercase
+/// Converts to uppercase, alias for the `|upper` filter
 ///
 /// ```
 /// # #[cfg(feature = "code-in-doc")] {
 /// # use rinja::Template;
 /// /// ```jinja
-/// /// <div>{{ word|upper }}</div>
+/// /// <div>{{ word|uppercase }}</div>
```

Comment:
I was about to send a PR for this one. Well done. :)

Reply:
Hehe, *great minds think alike* or something like that :D